tests/integration/case.js

```diff
@@ -0,0 +1,35 @@
+var assert = require('assert');
+var watchman = require('../../node');
+var client = new watchman.Client();
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+
+var platform = os.platform();
+if (platform == 'win32' || platform == 'darwin') {
```

Comment:
This is failing on windows right now... the reasons are a bit complex and out of scope of the task that you're working on, so for now, please just make this a 'darwin' specific test.

https://github.com/facebook/watchman/pull/321 will take of the windows side of this.